src/cmd/apply.go

```diff
@@ -344,6 +344,36 @@ func pushApps(list ...string) {
 				}
 				pushExtensions(app, subfilePath)
 			}
+			for _, assetFile := range manifestJson.Assets {
+				assetsList, err := filepath.Glob(filepath.Join(customAppPath, assetFile))
```

Comment:
We should add another case here where `assetList` returns an empty array, so we can rule out false positives when reporting issues.

Something like printing out a warning when length is 0.